doc/GuideForCommitters.md

```diff
@@ -0,0 +1,87 @@
+a checklist of items that committers must ensure have been completed prior to
+merging a pull request.
+
+
+## General Guidelines
+
+* Committers should not merge their own pull requests.
+
+* Do not merge a pull request if its title is prefixed with `WIP:` (indicating
+it's a work-in-progress)
+
+* If a review or feedback was requested by the author by `@mentioning` particular
+developers in a pull request, do not merge until all have provided the input
+requested.
+
+* Do not merge a pull request until all validation checks and continuous
+integration builds have passed.  If there are circumstances where you believe
+this rule should be violated (for example, a known test failure unrelated to
+this change) then the justification must be documented in the pull request
+prior to merging.
+
+* If a pull request is to revert a previous commit, ensure there is a reasonable
+explanation in the pull request from the author of the rationale along with a
+description of the problem if the commit is not reverted.
+
+
+## Pre-Merge Checklist
+
+* Ensure the pull request adheres to all the Eclipse OpenJ9 [Contribution Guidelines](https://github.com/eclipse/openj9/blob/master/CONTRIBUTING.md)
+
+* Ensure pull requests and issues are annotated with descriptive metadata by
+attaching GitHub labels.  The current set of labels can be found [here](https://github.com/eclipse/openj9/labels).
+
+* Be sure to validate the commit title and message on **each** commit in the PR (not
+just the pull request message) and ensure they describe the contents of the commit.
+
+* Ensure the commits in the pull request are squashed into only as few commits as
+is logically required to represent the changes contributed (in other words, superfluous
+commits are discouraged without justification).  This is difficult to quantify
+objectively, but the committer should verify that each commit contains distinct
+changes that should not otherwise be logically squashed with other commits in the
+same pull request.
+
+* When commits are pushed to a pull request, TravisCI builds launch automatically 
+to build the changes on x86 Linux (with and without the linter). However, as this only 
+builds the changes, you must initiate a pull request build on all affected platforms 
+prior to merging. To launch a pull request build, see [Triggering PR Builds](https://github.com/eclipse/openj9/tree/master/buildenv/jenkins).
+
+   If testing is only warranted on a subset of platforms (for example, only files
+built on x86 are modified) then pull request testing can be limited to only those
+platforms.  However, a comment from the committer providing justification is
+mandatory.
+
+   If newer commits have been added (either as truly new commits or a rebase of existing
+commits), PR builds must be triggered again.
+
+   If any builds fail because of infrastructure issues, they must be restarted.
+
+* Ensure the author has updated the copyright date on each file modified to the
```

Comment:
This should be covered by the copyright date checker which automatically runs on the PR.  Maybe link to the job?

Reply:
Given the heritage of this PR I think this is only a restriction on OMR which does not have the copyright date checker job. We should be able to remove this bullet as this should be covered by:

> Do not merge a pull request until all validation checks and continuous integration builds have passed.

from above list.